content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Instrumentation Libraries
+linkTitle: Instrumentation Libraries
```

Comment:
@cartermp - I think that for other SIGs we settled on "Libraries" as a short title, right?

```suggestion
linkTitle: Libraries
```